moped-editor/src/utils/taskOrderComponentHelpers.js

```diff
@@ -1,23 +1,5 @@
-import React from "react";
-import { TextField } from "@mui/material";
-
 /*
  * Socrata Endpoint
  */
 export const SOCRATA_ENDPOINT =
   "https://data.austintexas.gov/resource/ggav-ufvc.json?$limit=99999";
-
-/*
-/ Defines text input to render in MUI autocomplete
-*/
-export const TaskOrderInput = (params, error = false, variant = "standard") => {
```

Comment:
this was only used in the TaskOrderAutocomplete component that was only used in the SignalProjectTable